### PR TITLE
Fix with tesseract refactoring

### PR DESCRIPTION
**Pull request by qubka**

### CODE REVIEW

File: src/main/java/muramasa/antimatter/capability/machine/MachineRecipeHandler.java [hunk 1]
```diff
@@ -193,16 +191,12 @@ public void onMachineEvent(IMachineEvent event, Object... data) {
...
-                        this.tile.setMachineState(ACTIVE);
-                    break;
-                default:
-                    break;
+            if (event == MachineEvent.ENERGY_INPUTTED) {
+                if (this.tile.getMachineState() == IDLE)
+                    //NO_POWER is bad name i guess, by this i mean try to do a recipe check next tick.
+                    this.tile.setMachineState(NO_POWER);
+                if (this.tile.getMachineState() == POWER_LOSS)
+                    this.tile.setMachineState(ACTIVE);
```
Comment (Rongmario) [inline src/main/java/muramasa/antimatter/capability/machine/MachineRecipeHandler.java:199]: It's better to break here, as all the conditions would follow through + checked.

File: src/main/java/muramasa/antimatter/tile/TileEntityStorage.java [hunk 2]
```diff
@@ -27,8 +26,8 @@ public TileEntityStorage(Machine<?> type) {
     public void onLoad() {
         if (!energyHandler.isPresent() /*&& isServerSide()*/ && has(ENERGY)) energyHandler = Optional.of(new MachineEnergyHandler(this, 0, getMachineTier().getVoltage() * 64L, getMachineTier().getVoltage(), getMachineTier().getVoltage(), 1, 1){
             @Override
-            public boolean  canOutput(@Nonnull Dir direction) {
-                return tile.getOutputFacing().getIndex() == direction.getIndex();
+            public boolean  canOutput(Dir direction) {
+                return true;//tile.getOutputFacing().getIndex() == direction.getIndex();
```
Comment (Rongmario) [inline src/main/java/muramasa/antimatter/tile/TileEntityStorage.java:30]: Might as well fix the spacing

**Review by Rongmario — CHANGES_REQUESTED** [review]
Some minor changes, but looks fine overall.

**Comment by Rongmario** [comment]
Oh and please update latest tesseract in local.

File: src/main/java/muramasa/antimatter/capability/machine/MachineRecipeHandler.java (hunk 1 above)
Comment (qubka) [reply src/main/java/muramasa/antimatter/capability/machine/MachineRecipeHandler.java]: Done

**Pull request merged**